clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -118,8 +118,8 @@ public Selector(int maxReceiveSize, long connectionMaxIdleMs, Metrics metrics, T
         this.metricsPerConnection = metricsPerConnection;
     }
 
-    public Selector(long connectionMaxIdleMS, Metrics metrics, Time time, String metricGrpPrefix, Map<String, String> metricTags, ChannelBuilder channelBuilder) {
-        this(NetworkReceive.UNLIMITED, connectionMaxIdleMS, metrics, time, metricGrpPrefix, metricTags, true, channelBuilder);
+    public Selector(long connectionMaxIdleMS, Metrics metrics, Time time, String metricGrpPrefix, ChannelBuilder channelBuilder) {
+        this(NetworkReceive.UNLIMITED, connectionMaxIdleMS, metrics, time, metricGrpPrefix, metrics.config().tags(), true, channelBuilder);
```

Comment:
@junrao Ah, I didn't notice this comment yesterday night.

We probably need the default tags here. This default tags will be passed to `metricTags` in `Selector`, which is then used in `SelectorMetrics` to form sensor names. Alternatively, we can avoid passing defaults tags, but instead retrieve tags from `metrics.config().tags()` to form sensor names. I just think the current approach is cleaner. What do you think?